api/api.go

```diff
@@ -292,3 +303,30 @@ func (c *Client) Skip(language, slug string) error {
 
 	return errors.New(pe.Error)
 }
+
+// Status sends a request to exercism to fetch the user's
+// completion status for the given language track.
+func (c *Client) Status(language string) (*StatusInfo, error) {
+	c.XAPIHost = "http://localhost:9393"
```

Comment:
This is for testing on my local machine with the API serving mock data, and needs to be removed once this is in good enough shape and ready for merging.